scripts/openzfs-tracking.sh

```diff
@@ -221,13 +222,17 @@ do
 	ZFSONLINUX_REGEX="^(openzfs|illumos)+.*[ #]+$OPENZFS_ISSUE[ ,]+*.*"
 
 	# Commit exceptions reference this Linux commit for an OpenZFS issue.
-	EXCEPTION=$(grep -E "^$OPENZFS_ISSUE.+" "$ZFSONLINUX_EXCEPTIONS")
+	EXCEPTION=$(grep -E "^$OPENZFS_ISSUE\s" "$ZFSONLINUX_EXCEPTIONS")
 	if [ -n "$EXCEPTION" ]; then
 		EXCEPTION_HASH=$(echo $EXCEPTION | cut -f2 -d' ')
+		EXCEPTION_COMMENT=$(echo $EXCEPTION | cut -d' ' -f3-)
 		if [ "$EXCEPTION_HASH" == "-" ]; then
 			ZFSONLINUX_HASH="-"
 			ZFSONLINUX_STATUS=$STATUS_NONAPPLICABLE
 			ZFSONLINUX_STATUS_TEXT=$STATUS_NONAPPLICABLE_TEXT
+			if [ -n "$EXCEPTION_COMMENT" ]; then
+			OPENZFS_DESC="$OPENZFS_DESC</br><b>comment: $EXCEPTION_COMMENT</b>"
```

Comment:
That would work.  But I'm not keen on having 70 odd PRs open which I can't close.

Alternately, I was thinking we could just add a page on the wiki and scrape it for the exception information rather than using the checked in version.  We could just use `curl` to grab the raw page from Github.

hat would allow project members to easily update and maintain it.  And it would still be simple to parse.  We could just extend the existing format slightly so a `!` in the second column means the commit is applicable but not applied for the reason described in the comment.